_posts/2017-07-19-runners-driver.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Concordion | Runners and Drivers"
+heading: "Concordion Presentations"
+description: "Layering test code with Runners and Drivers"
+identifier: "concordion-presentations"
+author: "Nigel Charman"
+homepage: http://tutansblog.blogspot.co.nz/
+---
+
```

Comment:
Please add: "Designing well structured test code can lead to effective reuse of the application drivers across test suites, as well as open up their use for exploratory and maintenance scripting. It avoids lock-in to any specific test runner or driver, opening up the potential for migrating to new test runners or drivers."